packages/web-components/src/image/image.styles.ts

```diff
@@ -0,0 +1,78 @@
+import { css } from '@microsoft/fast-element';
+import {
+  borderRadiusLarge,
+  borderRadiusMedium,
+  borderRadiusSmall,
+  borderRadiusXLarge,
+  colorNeutralStroke2,
+  shadow4,
+  strokeWidthThin,
+} from '../theme/design-tokens.js';
+
+/** Image styles
+ *
+ * @public
+ */
+export const styles = css`
+  :host ::slotted(*) {
```

Comment:
general question here as I re-review - do we want to support * (all) slotted selectors here or just img/svg, or does this also actually work with `<picture></picture>`? Perhaps we should start with just `img`? Curious of others thoughts here. `*` selector is costly but if we're resolving any element it makes sense. In the case of `img`, do we just want `img` given that's what is returned in v9?

Reply:
I'm good with img.

Reply:
Do I write `::slotted(img)` ?